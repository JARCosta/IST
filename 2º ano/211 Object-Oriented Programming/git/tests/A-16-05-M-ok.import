PARTNER|S1|Toshiba|Tokyo, Japan
PARTNER|M1|Rohit Figueiredo|New Delhi, India
PARTNER|R1|Ant√≥nio Figueiredo|Lisboa
BATCH_S|ROLHA|R1|2|500
BATCH_S|VIDRO|M1|5|500
BATCH_S|SAL|S1|5|500
BATCH_M|GARRAFA|R1|55|100|0.1|ROLHA:1#VIDRO:10
BATCH_M|GARRAFA2|R1|50|100|0.1|ROLHA:1#VIDRO:10
