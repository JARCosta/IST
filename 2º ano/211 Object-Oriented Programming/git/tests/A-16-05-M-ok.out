Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Parceiros
7 - Gestão de Transacções
8 - Menu de Consultas
9 - Ver Saldo Global
0 - Sair
Escolha uma opção: Gestão de Transacções
1 - Visualizar
2 - Registar desagregação
3 - Registar venda
4 - Registar compra
5 - Receber pagamento de venda
0 - Sair
Escolha uma opção: Identificador do parceiro: Identificador do produto: Quantidade: Gestão de Transacções
1 - Visualizar
2 - Registar desagregação
3 - Registar venda
4 - Registar compra
5 - Receber pagamento de venda
0 - Sair
Escolha uma opção: Identificador do parceiro: Identificador do produto: Quantidade: Gestão de Transacções
1 - Visualizar
2 - Registar desagregação
3 - Registar venda
4 - Registar compra
5 - Receber pagamento de venda
0 - Sair
Escolha uma opção: Identificador da transacção: DESAGREGAÇÃO|0|S1|GARRAFA|4|12|12|0|ROLHA:4:8#VIDRO:40:200
Gestão de Transacções
1 - Visualizar
2 - Registar desagregação
3 - Registar venda
4 - Registar compra
5 - Receber pagamento de venda
0 - Sair
Escolha uma opção: Identificador da transacção: DESAGREGAÇÃO|1|M1|GARRAFA2|4|-8|0|0|ROLHA:4:8#VIDRO:40:200
Gestão de Transacções
1 - Visualizar
2 - Registar desagregação
3 - Registar venda
4 - Registar compra
5 - Receber pagamento de venda
0 - Sair
Escolha uma opção: Identificador da transacção: Gestão de Transacções
1 - Visualizar
2 - Registar desagregação
3 - Registar venda
4 - Registar compra
5 - Receber pagamento de venda
0 - Sair
Escolha uma opção: Identificador da transacção: Gestão de Transacções
1 - Visualizar
2 - Registar desagregação
3 - Registar venda
4 - Registar compra
5 - Receber pagamento de venda
0 - Sair
Escolha uma opção: Identificador da transacção: DESAGREGAÇÃO|0|S1|GARRAFA|4|12|12|0|ROLHA:4:8#VIDRO:40:200
Gestão de Transacções
1 - Visualizar
2 - Registar desagregação
3 - Registar venda
4 - Registar compra
5 - Receber pagamento de venda
0 - Sair
Escolha uma opção: Identificador da transacção: DESAGREGAÇÃO|1|M1|GARRAFA2|4|-8|0|0|ROLHA:4:8#VIDRO:40:200
Gestão de Transacções
1 - Visualizar
2 - Registar desagregação
3 - Registar venda
4 - Registar compra
5 - Receber pagamento de venda
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Parceiros
7 - Gestão de Transacções
8 - Menu de Consultas
9 - Ver Saldo Global
0 - Sair
Escolha uma opção: Gestão de Parceiros
1 - Mostrar parceiro
2 - Mostrar parceiros
3 - Registar parceiro
4 - Activa/desactivar notificações de um produto
5 - Mostrar compras com parceiro
6 - Mostrar vendas (e desagregações) com parceiro
0 - Sair
Escolha uma opção: M1|Rohit Figueiredo|New Delhi, India|NORMAL|0|0|0|0
R1|António Figueiredo|Lisboa|NORMAL|0|0|0|0
S1|Toshiba|Tokyo, Japan|NORMAL|120|0|0|0
Gestão de Parceiros
1 - Mostrar parceiro
2 - Mostrar parceiros
3 - Registar parceiro
4 - Activa/desactivar notificações de um produto
5 - Mostrar compras com parceiro
6 - Mostrar vendas (e desagregações) com parceiro
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Parceiros
7 - Gestão de Transacções
8 - Menu de Consultas
9 - Ver Saldo Global
0 - Sair
Escolha uma opção: 